### PR TITLE
[PropertyAccess] Add an example for setter with a dynamic property

### PR DESCRIPTION
**Pull request by devojifr**
The magic __set method can add a dynamic property.

### CODE REVIEW

File: components/property_access.rst [hunk 1]
```diff
@@ -324,10 +324,12 @@ can use setters, the magic ``__set()`` method or properties to set values::
     $propertyAccessor->setValue($person, 'firstName', 'Wouter');
     $propertyAccessor->setValue($person, 'lastName', 'de Jong'); // setLastName is called
     $propertyAccessor->setValue($person, 'children', [new Person()]); // __set is called
+    $propertyAccessor->setValue($person, 'role', 'contributor'); // A new property 'contributor' is set dynamically
```
Comment (OskarStark) [inline components/property_access.rst]:
```suggestion
    $propertyAccessor->setValue($person, 'role', 'contributor'); // A new property 'role' is set dynamically
```
?

Comment (devojifr) [reply components/property_access.rst]: You're right! Sorry for my mistake!

**Review by OskarStark — APPROVED** [review]

**Comment by xabbuh** [comment]
I wouldn’t consider creating properties at runtime a good practice. Thus I would rather not like to add this.

**Comment by OskarStark** [comment]
I agree, thanks for proposing!

**Pull request closed without merging**